dash_bio/component_factory/_manhattan.py

```diff
@@ -78,80 +78,76 @@ def ManhattanPlot(
-- showlegend (bool; optional): Boolean indicating whether legends should be
-    shown. (Default: True)
+    "scores", etc.)
+- title (string; default 'Manhattan Plot') The title of the graph.
+- showgrid (bool; default true): Boolean indicating whether gridlines
+    should be shown.
+- xlabel (string; optional): Label of the x axis.
+- ylabel: (string; default '-log10(p)'): Label of the y axis.
+- point_size (number; default 5): Size of the points of the Scatter
+    plot.
+- showlegend (bool; default true): Boolean indicating whether legends
+    should be shown.
 - col (string; optional): A string representing the color of the
     points of the Scatter plot. Can be in any color format accepted by
-    plotly_js graph_objs. (Default: None) Default = None
-- suggestiveline_value (bool/float; optional): A value which must
+    plotly_js graph_objs.
+- suggestiveline_value (bool | float; default 8): A value which must
     be False to deactivate the option, or a numerical value
     corresponding to the p-value at which the line should be drawn.
-    The line has no influence on the data points. (Default:
-    -np.log10(1e-8))
-- suggestiveline_color (string; optional): Color of the suggestive
-  line. (Default: "grey")
-- suggestiveline_width (number): Width of the suggestive
-        line. (Default: 2)
-- genomewideline_value (bool/float; optional): A boolean which must be
-    False to deactivate the option, or a numerical value corresponding
-    to the p-value above which the data points are considered
-    significant. (Default: -np.log10(5e-8))
-- genomewideline_color (string; optional): Color of the genome wide
+    The line has no influence on the data points.
+- suggestiveline_color (string; default 'grey'): Color of the suggestive
+  line.
+- suggestiveline_width (number; default 2): Width of the suggestive
+    line.
+- genomewideline_value (bool | float; default -log10(5e-8)): A boolean
+    which must be False to deactivate the option, or a numerical value
+    corresponding to the p-value above which the data points are
+    considered significant.
+- genomewideline_color (string; default 'red'): Color of the genome wide
     line. Can be in any color format accepted by plotly_js
-    graph_objs. (Default: "red")
-- genomewideline_width (number; optional): Width of the genome wide
-  line. (Default: 1)
-- highlight (bool; optional): turning on/off the highlighting of data points
-    considered significant. (Default: True)
-- highlight_color (string; optional): Color of the data points
+    graph_objs.
+- genomewideline_width (number; default 1): Width of the genome wide
+  line.
+- highlight (bool; default True): turning on/off the highlighting of
+    data points considered significant.
+- highlight_color (string; default 'red'): Color of the data points
     highlighted because they are significant Can be in any color
-    format accepted by plotly_js graph_objs. (Default: "red")
+    format accepted by plotly_js graph_objs.
```

Comment:
```suggestion
    format accepted by plotly's graph_objs.
```
alternatively, `plotly.graph_objs`, `plotly.js's graph_objs`, ... ?